### DISCUSSION TITLE
Unsteady Restart resets current time

### DISCUSSION THREAD

**Discussion by kursatyurt** [su2code/SU2#2071] [General]
When I restarted an unsteady problem, I noticed that the "Cur_Time" is not recovered.

```
|   Time_Iter|    Cur_Time|   Time_Step| 
|        3781|  1.9841e-04|  1.9841e-04| 
```

For instance, I have a pitching airfoil and I am calculating the effective angle of attack based on time  using 

```
a_0 + a_1 * sin(omega*t)
```
where a_0 is the mean angle of attack and a_1 is the amplitude of pitching. 

As pitching is calculated from `time_iter` and `dt` if one does not change `dt` it is not a problem, however, one loses the connection between time and pitching/plunging motion. (I just realize when I am writing this I can use `time_iter` to recover this information but I still want to keep discussion open)

What should be the expected behavior from the CFD software resetting time or continue?

**Comment by pcarruscag** [comment]
We could **report** the Current time as `restart_iter * dt + iter * dt` but we need to be careful with stoping conditions based on time.
Also, note that we don't have a "continue" button so starting from scratch is not wrong, we do not have a way to continue time averages from a restarted simulation for example. So resetting the time to 0 may create fewer expectations.

**Reply by kursatyurt** [reply]
@pcarruscag thank you very much for your input, especially since the time averages are kind of unique for SU2. May be it is a good start for me to take a look at what others do (so far I had experience with DLR TAU, might be better to take a look to OF, commercial solvers are more complicated I am not going to count them.).
